pygfx/helpers/_axes.py

```diff
@@ -9,10 +17,12 @@ class AxesHelper(Line):
 
     Parameters:
         size (float): The length of the lines (default 1).
-        thickness (float): the thickness of the lines (default 2 px).
+        thickness (float): The thickness of the lines (default 2 px).
+        arrow_start (float): Percentage where the arrow head should start (default 0.80).
```

Comment:
Actually this shouldn't be a parameter, the arrow head should just maintain an equilateral triangle when viewed from the side, so I can compute where it should start using arrow_radius. That will give the axes helper a consistent look.